db/migrate/20210527024130_add_isolate_name_to_batches.rb

```diff
@@ -0,0 +1,5 @@
+class AddIsolateNameToBatches < ActiveRecord::Migration
```

Comment:
I think it would be better to add this column in the create table migration file and delete this file at all. Is there any specific reason of why this column is being created in a different file other than its table creation?

Reply:
yeah!! Great!!